### PR TITLE
Fix: Basic sample tests - sequential is flaky

### PR DESCRIPTION
**Pull request by diegolovison**
**Description of your changes:**
Fix: https://github.com/kubeflow/pipelines/issues/11135

**Checklist:**
- [ x ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by hbelmiro** [comment]
@diegolovison you need to use `/rerun-all` to trigger the GH Actions tests.

File: test/sample-test/requirements.in [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
+papermill
 fire
 yamale
-kubernetes
-
-google-cloud-bigquery<3
-google-cloud-storage<2
-# TODO: remove deprecated dependency
--r sdk/python/requirements-deprecated.txt
+kubernetes
```
Comment (DharmitD) [inline test/sample-test/requirements.in:8]: fix EoF here.

File: test/sample-test/run_sample_test.py [hunk 2]
```diff
@@ -153,27 +152,17 @@ def check(self):
...
+            elapsed_time = (end_time - start_time).seconds
+            utils.add_junit_test(self._test_cases, 'job completion', succ,
+                                 'waiting for job completion failure',
+                                 elapsed_time)
+            print(f'Pipeline {"worked" if succ else "Failed"}. Elapsed time: {elapsed_time}s')
+
+            ###### Delete Job ######
+            #TODO: add deletion when the backend API offers the interface.
+
+            assert succ
```
Comment (DharmitD) [inline test/sample-test/run_sample_test.py:168]: fix EoF here.

**Comment by VaniHaripriya** [comment]
Looks good to me except EOF.

**Pull request merged**